relay-metrics/src/aggregation.rs

```diff
@@ -1378,28 +1378,6 @@ impl<T: Iterator<Item = Bucket>> FusedIterator for CappedBucketIter<T> {}
 ///
 /// Internally, the aggregator maintains a continuous flush cycle every 100ms. It guarantees that
 /// all elapsed buckets belonging to the same [`ProjectKey`] are flushed together.
-///
-/// Receivers must implement a handler for the [`FlushBuckets`] message:
```

Comment:
Removing the example below makes sense since it's now trivial, but I would keep this line.